Sources/ScipioKit/Producer/PIF/FrameworkBundleAssembler.swift

```diff
@@ -153,11 +153,96 @@ struct FrameworkBundleAssembler {
             )
         }
     }
+}
+
+extension FrameworkBundleAssembler {
+    struct ResourcesProcessor {
+        private let isVersionedBundle: Bool
+        private let sourceFrameworkBundlePath: TSCAbsolutePath
+        private let sourceFrameworkInfoPlistPath: TSCAbsolutePath
+        private let sourceResourceBundlePath: TSCAbsolutePath?
+        private let destinationFrameworkBundlePath: TSCAbsolutePath
```

Comment:
Can we avoid using `TSC` here? Can we convert it to the regular URL or `SwiftPM.Basic`

Reply:
`FileSystem` is in TSCBasic so file paths used for file system handling need to `TSCAbsolutePath` finally.

- https://github.com/swiftlang/swift-tools-support-core/blob/0cf39c086d7885cadb30073fd39133b0aeb18934/Sources/TSCBasic/FileSystem.swift#L177
- https://github.com/swiftlang/swift-package-manager/blob/0340bb12a56f9696b3966ad82c2aee1594135377/Sources/Basics/FileSystem/FileSystem%2BExtensions.swift#L27-L28

Is it good to hold these path as `Basics.AbsolutePath` and convert them to `TSCAbsolutePath` each time?

Reply:
I prefer to convert it at passing time.

Reply:
If you prefer that direction, let's handle it in a separate PR and replace all the references of  `TSCAbsolutePath` in the entire codebase.